packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialDetailOptions.ts

```diff
@@ -14,17 +14,17 @@ export class JsonLdOptionsCredentialStatus {
   public type!: string
 }
 
-export interface JsonLdOptionsRFC0593Options {
+export interface JsonLdCredentialDetailOptionsOptions {
```

Comment:
Is the double options here required? It looks a bit funky.

Reply:
The class is already called `JsonLdCredentialDetailOptions`, so yes it is needed. Gave it some thought but couldn't come up with a good solution and this is at least consistent